### PR TITLE
Visa immigration block

### PR DESCRIPTION
**Pull request by lsalivonaite-transform**
Added back the right to work/study and immigration status pages to this prototype.
Improved right to work/study question page, completely redone immigration status page to include preset answers in radio button form.

Added a blocking pattern for courses that do not sponsor visas to disallow users from applying if they don't have the right to work/study and require a visa sponsorship. 

Deleted a couple of unused pages.

Updated content on Find a course page to show the newest version.

### CODE REVIEW

File: app/views/details/personal-information/immigration.html [hunk 1]
```diff
@@ -2,30 +2,33 @@
...
+  <div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
   <h1 class="govuk-heading-l">{{ title }}</h1>
 
-  <p class="govuk-body">You’ll need the right to work or study in the UK for the duration of your course.</p>
-
-  <p class="govuk-body"><a href="https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-if-youre-a-non-uk-citizen#visa" class="govuk-link">Check if you’ve already got the right to work or study in the UK, or whether you’ll need to apply for a visa</a>.</p>
-
+  <p class="govuk-body">You can use a visa which is due to expire before the end of your course, as long as you have the right to renew it.</p>
+  <p class="govuk-body"><a href="#" class="govuk-link">Check your immigration status and whether you’ll need to apply for a visa</a>.</p>
```
Comment (MaeveRoseveare) [inline app/views/details/personal-information/immigration.html:20]: Need to add the link here?

**Review by MaeveRoseveare — COMMENTED** [review]
Looks ace!!

Is there a specific training provider I need to select to see the visa blocking pattern?

File: app/views/details/personal-information/immigration.html (hunk 1 above)
Comment (lsalivonaite-transform) [reply app/views/details/personal-information/immigration.html:20]: Yes, that needs to be added, but the previous link has been withdrawn.. So I dont know where to get a new link from for guidance about that

**Comment by lsalivonaite-transform** [comment]
> Looks ace!!
> 
> Is there a specific training provider I need to select to see the visa blocking pattern?

For now if you select 'No' to the right to work/study question you will be able to see the visa blocking pattern for any course/provider you select. It's because we don't have an API connected to this prototype

File: app/views/details/personal-information/immigration.html (hunk 1 above)
Comment (MaeveRoseveare) [reply app/views/details/personal-information/immigration.html:20]: Is it this: https://www.gov.uk/check-uk-visa (linked from Laura Tennant's doc)

**Comment by MaeveRoseveare** [comment]
> > Looks ace!!
> > Is there a specific training provider I need to select to see the visa blocking pattern?
> 
> For now if you select 'No' to the right to work/study question you will be able to see the visa blocking pattern for any course/provider you select. It's because we don't have an API connected to this prototype

Oh DOH! I'm being dumb. Ignore me

File: app/views/details/personal-information/immigration.html (hunk 1 above)
Comment (lsalivonaite-transform) [reply app/views/details/personal-information/immigration.html:20]: was about to comment the same link! I will add that

**Pull request closed without merging**